app/vmalert/main.go

```diff
@@ -198,6 +205,17 @@ func getAlertURLGenerator(externalURL *url.URL, externalAlertSource string, vali
 	}, nil
 }
 
+func checkConfigs() {
```

Comment:
function name looks missleading. It says `checkConfigs` but it is specifically used for `dryRun`. Since it is not reused anywhere - could we just use the code as is?